src/afl-fuzz-mutators.c

```diff
@@ -339,7 +339,7 @@ u8 trim_case_custom(afl_state_t *afl, struct queue_entry *q, u8 *in_buf,
 
   }
 
-  while (afl->stage_cur < afl->stage_max) {
+  while (afl->stage_cur <= afl->stage_max) {
```

Comment:
can someone have a look here? from our documentation we tell to return fir trim_post 1 if more trims can be done, resetting stage_max to 1, but stage_cur is 1 after the first loop ... what do?